Specs/Number/Arabic/NumberModel.json

```diff
@@ -2136,19 +2136,18 @@
     ]
   },
   {
-    "Input": "الإجابة هي ناقص صفر فاصلة مائة وخمسة وثلاثون",
```

Comment:
Same.

Reply:
The input was intended to be fraction but we figured that out there was some discrepancy so we reviewed with Language Expert, and got corrected them. Older form of input was not valid native sentence. That was integer (but still not in correct form). 
We have such integer cases.